R/slice.R

```diff
@@ -165,6 +174,93 @@ vec_as_index <- function(i, n, names = NULL) {
   vec_assert(n, integer(), 1L)
   .Call(vctrs_as_index, i, n, names)
 }
+#' @rdname vec_as_index
+#' @export
+vec_as_position <- function(i, n, names = NULL) {
+  if (is.object(i)) {
+    if (vec_is_coercible(i, int())) {
```

Comment:
I thought we were moving away from these abbreviations?

Reply:
I assumed from previous discussions that we were going to move them from rlang to vctrs?